course02/homework00/gavin/vector.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+#include <array>
+class VectorInt
+{
+public:
+	VectorInt();
+
+	void push_back(int element);
+
+	size_t size();
```

Comment:
`size_t` is C, not C++, please use `std::size_t`. This will compile on most, if not all, platform, but strictly speaking, it does not have to.